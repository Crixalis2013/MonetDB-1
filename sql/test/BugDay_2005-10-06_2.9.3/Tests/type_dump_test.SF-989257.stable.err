stderr of test 'type_dump_test.SF-989257` in directory 'sql/test/BugDay_2005-10-06_2.9.3` itself:


# 16:39:15 >  
# 16:39:15 >  "mclient" "-lsql" "-ftest" "-tnone" "-Eutf-8" "-i" "-e" "--host=/var/tmp/mtest-178400" "--port=31650"
# 16:39:15 >  


# 16:39:15 >  
# 16:39:15 >  "Done."
# 16:39:15 >  

