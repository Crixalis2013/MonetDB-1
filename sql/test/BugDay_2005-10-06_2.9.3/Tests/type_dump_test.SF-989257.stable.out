stdout of test 'type_dump_test.SF-989257` in directory 'sql/test/BugDay_2005-10-06_2.9.3` itself:


# 16:39:15 >  
# 16:39:15 >  "mclient" "-lsql" "-ftest" "-tnone" "-Eutf-8" "-i" "-e" "--host=/var/tmp/mtest-178400" "--port=31650"
# 16:39:15 >  

#CREATE TABLE voyages (
#	number mediumint,
#	number_sup character(1),
#	trip mediumint,
#	trip_sup character(1),
#	boatname varchar(50),
#	master varchar(50),
#	tonnage mediumint,
#	type_of_boat varchar(30),
#	built varchar(15),
#	bought varchar(15),
#	hired varchar(15),
#	yard character(1),
#	chamber character(1),
#	departure_date varchar(15),
#	departure_harbour varchar(30),
#	cape_arrival varchar(15),
#	cape_departure varchar(15),
#	arrival_date varchar(15),
#	arrival_harbour varchar(30),
#	next_voyage mediumint,
#	particulars varchar(507)
#select c.name, c.type, c.type_digits, c.type_scale, c."null", c."default", c.number from sys._columns c, sys._tables t, sys.schemas s
#where c.table_id = t.id and t.name = 'voyages' and t.schema_id = s.id and s.name = 'sys' order by c.number;
% sys.c,	sys.c,	sys.c,	sys.c,	sys.c,	sys.c,	sys.c # table_name
% name,	type,	type_digits,	type_scale,	null,	default,	number # name
% varchar,	varchar,	int,	int,	boolean,	varchar,	int # type
% 17,	7,	3,	1,	5,	0,	2 # length
[ "number",	"int",	32,	0,	true,	NULL,	0	]
[ "number_sup",	"char",	1,	0,	true,	NULL,	1	]
[ "trip",	"int",	32,	0,	true,	NULL,	2	]
[ "trip_sup",	"char",	1,	0,	true,	NULL,	3	]
[ "boatname",	"varchar",	50,	0,	true,	NULL,	4	]
[ "master",	"varchar",	50,	0,	true,	NULL,	5	]
[ "tonnage",	"int",	32,	0,	true,	NULL,	6	]
[ "type_of_boat",	"varchar",	30,	0,	true,	NULL,	7	]
[ "built",	"varchar",	15,	0,	true,	NULL,	8	]
[ "bought",	"varchar",	15,	0,	true,	NULL,	9	]
[ "hired",	"varchar",	15,	0,	true,	NULL,	10	]
[ "yard",	"char",	1,	0,	true,	NULL,	11	]
[ "chamber",	"char",	1,	0,	true,	NULL,	12	]
[ "departure_date",	"varchar",	15,	0,	true,	NULL,	13	]
[ "departure_harbour",	"varchar",	30,	0,	true,	NULL,	14	]
[ "cape_arrival",	"varchar",	15,	0,	true,	NULL,	15	]
[ "cape_departure",	"varchar",	15,	0,	true,	NULL,	16	]
[ "arrival_date",	"varchar",	15,	0,	true,	NULL,	17	]
[ "arrival_harbour",	"varchar",	30,	0,	true,	NULL,	18	]
[ "next_voyage",	"int",	32,	0,	true,	NULL,	19	]
[ "particulars",	"varchar",	507,	0,	true,	NULL,	20	]
#drop table voyages;

# 16:39:15 >  
# 16:39:15 >  "Done."
# 16:39:15 >  

